spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JGitEnvironmentRepository.java

```diff
@@ -205,6 +205,10 @@ public void setGitCredentialsProviderFactory(GitCredentialsProviderFactory gitCr
 		this.gitCredentialsProviderFactory = gitCredentialsProviderFactory;
 	}
 
+	public GitCredentialsProviderFactory getGitCredentialsProviderFactory() {
```

Comment:
I dont think this needs to be public if it is just used for tests

Reply:
I saw this practice in other cases here - like the  MultipleJGitEnvironmentRepository.getRepos() - it was used only in tests before my change.

What do you suggest instead?

Reply:
I think if its just used in tests right now there is no need to make it `public`

Reply:
so to change it to what?  package level?

Reply:
Yup, that is what I was thinking.

Reply:
done

Reply:
done. For some reason tests failed. How do I run them again?